### PR TITLE
refactor: alias new files

### PR DESCRIPTION
**Pull request by MaskDuck**
## Summary

title 

btw this has also fixed an typo in alias_license.py

## Checklist

- [ ] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
  - [ ] I have run `task pyright` and fixed the relevant issues.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by EmreTech — APPROVED** [review]

File: discord/bans.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+DEALINGS IN THE SOFTWARE.
+
+--------------
+Aliased module. See the same file in the nextcord folder for more information
+Autogenerated by aliasgen.py
+"""
+from nextcord.bans import TYPE_CHECKING, BanEntry, NamedTuple, Optional, annotations
+
```
Comment (TAG-Epic) [inline discord/bans.py:30]: This doesnt seem like valid syntax

**Review by TAG-Epic — CHANGES_REQUESTED** [review]
There is also a big chance voice was broken by this as that has to be fixed manually

**Comment by ooliver1** [comment]
Please can you actually test your own PRs

**Comment by DenverCoder1** [comment]
This PR doesn't fix all of the files.

It probably needs to be run again after updating the base branch.

When this PR was opened, the aliasgen script didn't work and was crashing part-way through because of the issue that was fixed in #888.

**Pull request closed without merging**